content/en/ecosystem/vendors.md

```diff
@@ -18,7 +18,7 @@ OpenTelemetry in their commercial products.
 | AWS                        | Yes               | No          | [aws-otel.github.io](https://aws-otel.github.io)                                                                                                      |
 | Azure                      | Yes               | No          | [docs.microsoft.com/...](https://docs.microsoft.com/azure/azure-monitor/app/opentelemetry-overview)                                                   |
 | Coralogix                  | Yes               | Yes         | [coralogix.com/...](https://coralogix.com/docs/opentelemetry/)                                                                                        |
-| DaoCloud                   | Yes               | Yes         | [docs.daocloud.io/...](https://docs.daocloud.io/en/insight/06UserGuide/01quickstart/otel/otel/)                                                       |
+| DaoCloud                   | Yes               | Yes         | [docs.daocloud.io/...](https://docs.daocloud.io/en/insight/03ProductBrief/WhatisInsight/)                                                       |
```

Comment:
As @cartermp mentioned elsewhere, we prefer links into the OTel docs, so the original link was better.